test/versioned-external/checkout-external-tests.js

```diff
@@ -11,7 +11,7 @@
 // 'rm' not available in Node 12 but considered deprecated in newer versions
 // 'fs/promises' not available in Node 12
 const { existsSync } = require('fs')
-const { rmdir, mkdir } = require('fs').promises
+const { rm, mkdir } = require('fs').promises
```

Comment:
I don't think this has been completely addressed.  @michaelgoin mentioned fs/promises not available in node 12.  not sure what he wanted to use in there.  If this is ok you forgot to remove the comments above

Reply:
Resolving; I'll remove this comment, which is worded in a way that suggests the opposite of its intent.